scripts/oneoff/export-dimension-titles-as-csv.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+
+import sys
+import csv
+
+from application import db
+from application.factory import create_app
+from application.config import DevConfig
+
+sys.path.insert(0, ".")
+
+
+def _calculate_short_title(page_title, dimension_title):
+    # Case 1 - try stripping the dimension title
+    low_title = dimension_title.lower()
+    if low_title.find(page_title.lower()) == 0:
+        return dimension_title[len(page_title) + 1 :]
+
+    # Case 2 - try cutting using the last by
+    by_pos = dimension_title.rfind("by")
```

Comment:
Possibly worth doing `rfind(" by ")` (including a space before and afterward), to make sure we're finding the word by itself as opposed to any suffix or mid-word string?

Reply:
I literally just copied this from the existing method in `application/dashboard/data_helpers.py` – will import it instead.